edgelet/edgelet-http/src/error.rs

```diff
@@ -81,18 +90,27 @@ pub enum ErrorKind {
     #[fail(display = "An error occurred with the proxy {}", _0)]
     Proxy(Uri),
 
+    #[fail(
+        display = "Preparing a PCKS12 client certificate identity failed {}",
+        _0
+    )]
+    PKCS12Identity(String),
+
     #[fail(display = "An error occurred in the service")]
     ServiceError,
 
-    #[fail(display = "An error occured configuring the TLS stack")]
+    #[fail(display = "An error occurred configuring the TLS stack")]
     TlsBootstrapError,
 
-    #[fail(display = "An error occured during creation of the TLS identity from cert")]
+    #[fail(display = "An error occurred during creation of the TLS identity from cert")]
     TlsIdentityCreationError,
 
     #[fail(display = "Token source error")]
     TokenSource,
 
+    #[fail(display = "Trust bundle usage error")]
```

Comment:
Given the circumstances in how this is used, perhaps `"Could not parse trust bundle"`